chromeshack.css

```diff
@@ -788,6 +869,54 @@ a.refresh_pending {
     height: 15px !important;
 }
 
+.postmeta a[id^="get_lol_users_"] {
```

Comment:
Any concern on browser performance with selectors like these?  If it's our own elements, why not stick a class on it?